### PR TITLE
refactor(android): Move log controls to Logs section of Settings

### LINKED ISSUE #2726: android: Move log controls to "Logs" section of Settings

### PR DESCRIPTION
**Pull request by jasonboukheir**
Fixes #2726

### CODE REVIEW

**Comment by jasonboukheir** [comment]
I'm having some trouble generating log files from the app to test out the log settings. Is there a recipe for logs to generate on Android?

File: kotlin/android/app/src/main/java/dev/firezone/android/features/settings/ui/LogSettingsFragment.kt [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+    private fun setupStateObservers() {
+        lifecycleScope.launch {
+            repeatOnLifecycle(Lifecycle.State.STARTED) {
+                viewModel.uiState.collect { uiState ->
+                    with(binding) {
+                        val hasLogs = uiState.logSizeBytes > 0
+                        btShareLog.isEnabled = hasLogs
+                        btClearLog.isEnabled = hasLogs
+                        val logSize = "${uiState.logSizeBytes / 1000L} KB"
```
Comment (jamilbk) [inline kotlin/android/app/src/main/java/dev/firezone/android/features/settings/ui/LogSettingsFragment.kt]:
```suggestion
                        val logSize = "${uiState.logSizeBytes / 1000000L} MB"
```

File: kotlin/android/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -4,12 +4,14 @@
 	<!-- Settings -->
 	<string name="settings_title">Settings</string>
 	<string name="advanced_settings_title">Advanced</string>
+	<string name="log_settings_title">Logs</string>
```
Comment (jamilbk) [inline kotlin/android/app/src/main/res/values/strings.xml]:
```suggestion
	<string name="log_settings_title">Diagnostic Logs</string>
```

**Review by jamilbk — CHANGES_REQUESTED** [review]
Just some minor UI things otherwise LGTM

**Review by jamilbk — APPROVED** [review]

**Pull request merged**